src/py/flwr/cli/new/templates/main.pytorch.py.tpl

```diff
@@ -0,0 +1 @@
+"""$project_name: A Flower / PyTorch app."""
```

Comment:
I like the text!

Instead of `main.py`, we need `server.py` and `client.py`. Or we add both `ServerApp` and `ClientApp` to `main.py`.

Reply:
I'll make the adjustment.